r/readable-stream/readable-stream_v2.3.7_rhel_8.5.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -e
+
+# -----------------------------------------------------------------------------
+#
+# Package           : readable-stream
+# Version           : v2.3.7
```

Comment:
same thing on this one and all others @bhagat-singh001 , if its not specific to a version, let's remove it from the script name, thereby making it more generic.

Reply:
Removed version from the filename .